index.html

```diff
@@ -2,498 +2,406 @@
-                <h3>Language</h3>
-              </li>
-            </ul>
-            <ul class="lang">
-              <li>Javascript</li>
-              <li>Ruby</li>
-              <li>Html</li>
-              <li>Css</li>
-            </ul>
-          </li>
-          <li class="list">
-            <ul class="title">
-              <li><i class="fas fa-square"></i></li>
-              <li>
-                <h3>Frameworks</h3>
-              </li>
-            </ul>
-            <ul class="lang">
-              <li>Bootstrap</li>
-              <li>Ruby on Rails</li>
-              <li>RSPec</li>
-              <li>CapyBara</li>
-              <li>Selenium</li>
-            </ul>
-          </li>
-          <li class="list">
-            <ul class="title">
-              <li><i class="fas fa-circle"></i></li>
-              <li>
-                <h3>Skills</h3>
-              </li>
-            </ul>
-            <ul class="lang">
-              <li>Codekit</li>
-              <li>Github</li>
-              <li>Codepen</li>
-              <li>Gitlab</li>
-              <li>Terminal</li>
-            </ul>
-          </li>
-        </ul>
-      </div>
-    </section>
-    <footer class="footer" id="contact-me-page">
-      <section class="contact">
-        <div class="title-box">
-          <h3 class="contact-message">
-            I'm always interested in hearing about new projects, so if you'd
-            like to chat please get in touch.
-          </h3>
-        </div>
-        <form class="contact-form" action="https://formspree.io/f/xdoyzdjj" method="POST">
-          <ul>
-            <li class="name-email">
-              <label for="name"></label>
-              <input type="text" id="name" name="user_name" placeholder="Full Name" maxlength="30" required />
-            </li>
-            <li class="name-email">
-              <label for="email"></label>
-              <input type="email" id="email" name="user-email" placeholder="Email address" required />
-            </li>
-            <li class="text-area">
-              <label for="msg"></label>
-              <textarea id="msg" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
-            </li>
-            <li>
-              <button class="contact-btn" type="submit">Get in touch</button>
-            </li>
-          </ul>
-        </form>
-      </section>
-      <section class="footer-icons-container">
-        <ul class="footer-icons">
-          <li><a href="#"><i class="fab fa-github fa-lg"></i></a></li>
-          <li><a href="#"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
-          <li><a href="#"><i class="fab fa-angellist fa-lg"></i></a></li>
-          <li><a href="#"><i class="fab fa-twitter fa-lg"></i></a></li>
-          <li><a href="#"><i class="fab fa-medium-m fa-lg"></i></a></li>
-        </ul>
-      </section>
-    </footer>
-  </div>
-  <script src="./script.js"></script>
+	<div id="overlay"></div>
```

Comment:
- [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.
- [ ] The popup is responsive, it fits mobile and desktop sizes properly.
- [  ] The popup follows the design for both desktop and mobile version from the chosen template in Figma (e.g., font, colors, images, tests, margins).

Reply:
Thank you for the review. But I'm not sure you've mentioned what is not working properly. You only mentioned what is working, right?

Could you elaborate on the problem/s? 
Thank you